crates/node/platform/src/actions/run_target.rs

```diff
@@ -77,6 +79,60 @@ fn create_node_options(
     Ok(options)
 }
 
+fn find_package_bin(
+    command: &mut Command,
+    node_options: &[String],
+    starting_dir: &Path,
+    working_dir: &Path,
+    bin_name: &str,
+) -> Result<Option<Command>, ToolError> {
+    let possible_bin_path = match node::find_package_bin(starting_dir, bin_name)? {
+        Some(bin) => Ok(bin),
+        None => Err(ToolError::MissingBinary(bin_name.to_owned())),
+    };
+
+    match possible_bin_path? {
+        // Rust, Go
+        BinFile::Binary(bin_path) => {
+            return Ok(Some(Command::new(bin_path)));
+        }
```

Comment:
I saw this in the original code. Why is this needed in the node package?

Reply:
@jpoz Node (npm) packages must be executed with the node binary. Some npm packages ship go/rust code (like us), and trying to pass it to the node binary crashes, so we replace the node binary entirely with the go/rust one.